perfkitbenchmarker/packages/sysbench05plus.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing sysbench 0.5 (and later) installation and cleanup logic.
```

Comment:
hmm... I looked at all other packages in this directory, no one provided uninstallation functions. Actually I had the same question about uninstallation, after chatting with Eric, he said it's not necessary here, the package manager will capture a package baseline before the benchmark run, and restore the static VM to the baseline after the run, so individual package installer does not need to worry about uninstallation, which is nice.